### PR TITLE
Add changeDeposit

### PR DESCRIPTION
**Pull request by makoto**
Allows owner to change deposit while no one has registered.

This PR is dependent on smart contract [PR](https://github.com/wearekickback/contracts/pull/23). No server PR is required.

### CODE REVIEW

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr149-app-wearekickback.surge.sh

File: src/components/SingleEvent/Admin/ChangeDeposit.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+              analyticsId="Change Deposit"
+              onClick={() =>
+                changeDeposit({
+                  variables: {
+                    address,
+                    deposit: text
+                  }
+                })
+              }
+              disabled={numParticipants >= 0}
```
Comment (hiddentao) [inline src/components/SingleEvent/Admin/ChangeDeposit.js]:
```suggestion
              disabled={numParticipants >= 1}
```

File: src/components/SingleEvent/Admin/ChangeDeposit.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  margin-bottom: 10px;
+`
+
+const ChangeDeposit = ({ address, currentDeposit, numParticipants }) => {
+  const depositToDisplay = toEthVal(currentDeposit)
+    .toEth()
+    .toFixed(2)
+
+  const [text, setText] = useState('')
+  return (
```
Comment (hiddentao) [inline src/components/SingleEvent/Admin/ChangeDeposit.js]: Should state somewhere that amount should be in ETH

File: src/components/SingleEvent/Admin/ChangeDeposit.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react'
+import styled from 'react-emotion'
+import { Mutation } from 'react-apollo'
+import gql from 'graphql-tag'
+
+import Button from '../../Forms/Button'
+import DefaultTextInput from '../../Forms/TextInput'
+import { toEthVal } from '../../../utils/units'
+
+const CHANGE_DEPOSIT = gql`
```
Comment (hiddentao) [inline src/components/SingleEvent/Admin/ChangeDeposit.js]: Move this query into `graphql/mutations.js`

File: src/components/SingleEvent/Admin/ChangeDeposit.js (hunk 1 above)
Comment (makoto) [reply src/components/SingleEvent/Admin/ChangeDeposit.js]: Good catch. Thanks

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr149-app-wearekickback.surge.sh

**Comment by makoto** [comment]
Aleardy merged into dai integration branch

**Pull request closed without merging**